android/platform.xml

```diff
@@ -4,4 +4,11 @@
         <packagedDependency>unityads.jar</packagedDependency>
     </packagedDependencies>
     
+
+    <packagedResources> 
+  <packagedResource> 
```

Comment:
you can also fix the spacing here while you are making the other changes :stuck_out_tongue: